test/integration/transactionsAPITests.coffee

```diff
@@ -153,8 +164,8 @@ describe "API Integration Tests", ->
         td = JSON.parse JSON.stringify transactionData
         td.channelID = channel._id
         td.request.body = ''
-        td.response.body = ''
-        td.response.body += '1234567890' for i in [0...2*1024*1024]
+        respBody = largeBody
+        td.response.body = respBody
```

Comment:
I think we can just use 'largeBody' variable here and remove the variable declaration above